Unsafe assignment of an error typed value  @typescript-eslint/no-unsafe-assignment

This module seems to cause a problem for typescript linting. 


```
import { Component } from '@angular/core';
import { RouterOutlet } from '@angular/router';
import {ControlComponent,  MapComponent} from 'ngx-mapbox-gl';
import {I18NextService} from 'angular-i18next';
import i18next from 'i18next';

@Component({
  selector: 'app-root',
  imports: [RouterOutlet, MapComponent, ControlComponent],
  templateUrl: './app.component.html',
  styleUrl: './app.component.css'
})
export class AppComponent {
  title = 'MapBoxDemo';
  constructor(private i18ns: I18NextService) {
    const message = i18ns.t("blub"); // <-- this line causes an unsafe assignment
    console.log( message)
    const s = i18next.t("hello");// <-- using i18next directly does not cause an unsafe assigment
    console.log(s);
  }
}

```

Reproducer: clone
https://github.com/tg-freigmbh/angular-i18-reproducer
run 
`ng lint`
This will result in something like
```
G:\development\reproducers\MapBoxDemo\src\app\app.component.ts
  16:11  error  Unsafe assignment of an error typed value  @typescript-eslint/no-unsafe-assignment
```
This can be worked around by changing 
`    "moduleResolution": "bundler", `
to 
`    "moduleResolution": "node", `
(But bundler should be working for modern angular applications)

--- a/libs/angular-i18next/src/lib/models.ts
+++ b/libs/angular-i18next/src/lib/models.ts
@@ -1,12 +1,13 @@
 import type * as i18n from 'i18next';
-import type { $Dictionary as I18Next$Dictionary } from 'i18next/typescript/helpers';
 
 export type FormatPipeOptions = { format?: string; lng?: string; case?: string; [key: string]: any };
 export type PrependPipeOptions = {
   prependScope?: boolean;
   prependNamespace?: boolean;
 };
-export type PipeOptions = i18n.TOptions<I18Next$Dictionary> & { defaultValue?: string; } &
+
+
+export type PipeOptions = i18n.TOptions &
   FormatPipeOptions &
   PrependPipeOptions;
 

--- a/libs/angular-i18next/src/lib/services/translation.service.ts
+++ b/libs/angular-i18next/src/lib/services/translation.service.ts
@@ -1,11 +1,10 @@
 import * as i18n from 'i18next';
-import { $Dictionary as I18Next$Dictionary } from 'i18next/typescript/helpers';
 import { I18NextLoadResult } from '../I18NextLoadResult';
 import { ITranslationEvents } from './translation.events';
 
 type Modify<T, R> = Omit<T, keyof R> & R;
 
-export type ITranslationOptions = i18n.TOptions<I18Next$Dictionary>  & { defaultValue?: string; };
+export type ITranslationOptions = i18n.TOptions
 
 export type ITranslationService = Modify<Partial<i18n.i18n>, {
 

--- a/tsconfig.base.json
+++ b/tsconfig.base.json
@@ -4,7 +4,7 @@
     "rootDir": ".",
     "sourceMap": true,
     "declaration": false,
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
     "alwaysStrict": true,
     "forceConsistentCasingInFileNames": true,
     "downlevelIteration": true,

